package.json

```diff
@@ -34,7 +34,7 @@
     "@glimmer/tracking": "^1.0.3",
     "broccoli-funnel": "^3.0.1",
     "broccoli-merge-trees": "^4.1.0",
-    "denali-css": "^2.3.0",
+    "denali-css": "^2.3.2",
```

Comment:
`~`?

Reply:
I am hoping the 2.3.x issues was just a one off. Typically, floating the minor has works for us.